x-pack/metricbeat/module/sql/_meta/docs.asciidoc

```diff
@@ -363,15 +352,175 @@ For example:
   period: 10s
   hosts: ["postgres://postgres:postgres@localhost:5432/stuff?sslmode=disable"]
   driver: "postgres"
-  sql_query: "SELECT datid, datname, blks_read, blks_hit, tup_returned, tup_fetched, stats_reset FROM pg_stat_database"
-  sql_response_format: table
+  raw_data.enabled: true
 
-- module: sql
-  metricsets:
-    - query
-  period: 10s
-  hosts: ["postgres://postgres:postgres@localhost:5432/stuff?sslmode=disable"]
-  driver: "postgres"
-  sql_query: "SELECT * FROM pg_catalog.pg_tables pt WHERE schemaname ='pg_catalog'"
-  sql_response_format: table
+  sql_queries:
+    - query: "SELECT * FROM pg_catalog.pg_tables pt WHERE schemaname ='pg_catalog'"
+      response_format: table
+
+    - query: "SELECT datname, datid FROM pg_stat_database;"
+      response_format: variables
+----
+
+The example generates this event: The response event is generated in new format by enabling the flag `raw_data.enabled`.
+
+[source,json]
+----
+{
```

Comment:
This is an elasticsearch document again.

Reply:
done